### PR TITLE
docs(README): Updated Documentation for Repository ID's in Installation Authentication Section 

### PR DESCRIPTION
**Pull request by NickLiffen**
- Fixes 1 our of 2 problems found in #277 👍 updated README to show correct information 💯

### CODE REVIEW

**Comment by NickLiffen** [comment]
@gr2m did I do something wrong with the `project-board` CI check? 🤔

**Comment by wolfy1339** [comment]
You did nothing wrong. That check is for maintainers, as it sorts issues by us setting labels

**Comment by NickLiffen** [comment]
> You did nothing wrong. That check is for maintainers, as it sorts issues by us setting labels

Ah perfect 👍 Okay thank you 👍 Appreciate that just wanted to check 💯 

I will raise another PR for the other issue raised in #277 later this week, just wanted to get this one done 👍 

Thanks for all your help.

File: README.md [hunk 1]
```diff
@@ -556,10 +556,10 @@ Create, reset, refresh, delete OAuth user-to-server tokens
         <code>repositoryIds</code>
       </th>
       <th>
-        <code>array of string</code>
+        <code>array of numbers</code>
       </th>
       <td>
-        The `id`s of the repositories that the installation token can access.
+        The <code>id</code> of the repositories that the installation token can access. Also known as a <code>databaseID</code> when querying the repository object in GitHub's v4 API.
```
Comment (gr2m) [inline README.md:562]: The v3/v4 terminologies are deprecated, we refer to them as REST API and GraphQL API now.

```suggestion
        The <code>id</code> of the repositories that the installation token can access. Also known as a <code>databaseID</code> when querying the repository object in GitHub's GraphQL API.
```

**Review by gr2m — APPROVED** [review]
Thanks a lot!

File: README.md (hunk 1 above)
Comment (NickLiffen) [reply README.md]: Good to know 🎧 ❤️

**Pull request merged**